src/arrays.js

```diff
@@ -1,61 +1,61 @@
 const getNthElement = (index, array) => {
-  // your code here
-};
-
-const arrayToCSVString = (array) => {
-  // your code here
+  if (array.length <= 4) {
```

Comment:
I'm not sure why you have an `if` statement here. `return array[index]` will work for any length array.